orchestration/OrchestrationService/ValidationResourceGroupSetup.ps1

```diff
@@ -0,0 +1,103 @@
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory=$false)]
+    [string]
+    $ResourceGroupName,
+    [Parameter(Mandatory=$false)]
+    [string]
+    $ResourceGroupLocation,
+    [Parameter(Mandatory=$false)]
+    [switch]
+    $TearDown
+)
+
+$defaultResourceGroupName = "vdc-validation-rg";
```

Comment:
Here you also have a default resource group name. I think this script should be agnostic on the name, or make New-Deployment agnostic of the name, the idea is to have the default name in one place only

Reply:
Removed from New-Deployment. Instead, importing the validation resource group setup script which will expose the variable globally.